### ISSUE TITLE
Environment Variables are being ignored (0.7.3)

### ISSUE THREAD

**Issue by lastobelus** [Shopify/themekit#462]
### What I expected to happen
When I set `THEMEKIT_PASSWORD` & `THEMEKIT_STORE` environment variables correctly
And then I run `theme download`
Themekit should download theme files

### What actually happened
themekit outputs:
```
Could not load any valid environments
```

I know the env variables are set correctly because
```
theme download -p $THEMEKIT_PASSWORD -s $THEMEKIT_STORE
```
works as expected.

### My Environment

OS (including version): OS X 10.12.6
Themekit version: ThemeKit 0.7.3 darwin/amd64

**Comment by tanema** [comment]
This is because right now all environments are loaded and Environment variables have changed some-what. This may have to change in the near future though and I will be in touch on this issue.

**Comment by tanema** [comment]
This has been fixed in 0.7.4 just released.
